### DISCUSSION TITLE
docker cli import exsiting libraries failed,file not exsit

### DISCUSSION THREAD

**Discussion by Anrufen** [immich-app/immich#3376] [Q&A] [answered]
Hello, I use docker cli import exsiting photos failed

Here is my settings

docker-compose.yml /path/to/media is

```
/mnt/photos:/mnt/photos
```

user's External Path is
```
/mnt/photos
```

creat alias

```
alias immich='docker run -it --rm -v "/mnt/photos:/import" ghcr.monlor.com/immich-app/immich-cli:latest'
```

Then

```
immich upload --key abc123 --server http://192.168.1.14:2283/api --recursive /import --import
```

All things right, but when I choose yes, It's print `file is not exsit`

Which step or setting is wrong? Thanks!

**Comment by bo0tzz** [comment, marked as answer]
You need to mount the files to `/mnt/photos` inside of the CLI container as well, rather than to `/import` like you are doing now.

**Reply by Anrufen** [reply]
Works for me, Thank you very much!

Btw,why immich not support -v the existing photo dir directly like photoprism instead of import?
